app/build.gradle

```
@@ -1,6 +1,9 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+
+    //
```

```suggestion
    // secrets-gradle-plugin
```

Globalの方で書いてるならこっちも書こう